azurerm/internal/services/frontdoor/migration/web_application_firewall_policy.go

```diff
@@ -111,6 +111,10 @@ func WebApplicationFirewallPolicyV0Schema() *schema.Resource {
 									"match_values": {
 										Type:     schema.TypeList,
 										Required: true,
+										Elem: &schema.Schema{
+											Type:         schema.TypeString,
+											ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
we can remove the validate from this since this is a schema migration, it's not needed